### PR TITLE
Add 4 new employees and update FAQs

### PR DESCRIPTION
**Pull request by ekumlin**
- Add new employers
    - Fidelity
    - Groupon
    - LA Times
    - Royal Caribbean
- Update sample file to make it "more fake"
- Update FAQs re: furloughs vs. layoffs
- Fix some logos and an error in Cinemark's data

### CODE REVIEW

File: public/employers/groupon.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+summary: Groupon implemented a number of useful cost-cutting measures, such as reducing marketing initiatives, but also resorted to furloughs while not reducing cash compensation for executive board members.
+citations:
+  - summary: Groupon let go of combined total of 2,800 workers&mdash;40% of their workforce&mdash;through layoffs and furloughs.
+    positivity: -1
+    type: publication
+    sources:
+      - source: The Motley Fool
+        link: https://web.archive.org/web/20200423022246/https://www.fool.com/investing/2020/04/15/groupon-to-lay-off-or-furlough-2800-workers-initia.aspx
+        date: 2020-04-15T13:09:00Z
+  - summary: Groupon implemented a hiring freeze, reduced marketing initiatives, and other cost-cutting measures.
```
Comment (frosteeze) [inline public/employers/groupon.yml:28]: I think this should be a 0 as it doesn't really benefit their employees too much.

File: public/employers/groupon.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    positivity: 1
+    type: publication
+    sources:
+      - source: The Motley Fool
+        link: https://web.archive.org/web/20200423022246/https://www.fool.com/investing/2020/04/15/groupon-to-lay-off-or-furlough-2800-workers-initia.aspx
+        date: 2020-04-15T13:09:00Z
+      - source: CNBC
+        link: https://web.archive.org/web/20200426010552/https://www.cnbc.com/2020/03/13/workforce-wire-coronavirus-heres-what-every-major-company-is-doing-about-the-pandemic.html
```
Comment (frosteeze) [inline public/employers/groupon.yml:36]: Date for this is MAR 13 20205:17 PM EDT

Comment (ekumlin) [reply public/employers/groupon.yml:36]: The initial post is, but they're updating this page constantly. As a result I don't know when the information was added for this particular company, so I'm not specifying a particular date for this one.

File: public/employers/groupon.yml (hunk 1 above)
Comment (ekumlin) [reply public/employers/groupon.yml:28]: That's fair. It may result in preventing layoffs, but since they did that anyway, maybe a 0 is fair.

**Pull request merged**